### PR TITLE
Release 0.7.0

### PR DESCRIPTION
**Pull request by RobAtticus**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,48 @@
...
+## 0.7.0 (2017-11-21)
+
+**High-level changes**
+* **Initial PostgreSQL 10 support**. TimescaleDB now should work on both PostgreSQL 9.6 and 10. As this is our first release supporting PG10, we look forward to community feedback and testing.
+* Support for `CLUSTER` on hypertables to recursively apply to chunks
+* Improve constraint handling of edge cases for `DATE` and `TIMESTAMP`
+* Fix `range_start` and `range_end` to properly handle the full 32-bit int space
+* Allow users to specify their desired partitioning function
+* Enforce that the time column contains not NULL values
+* Add testing infrastrcture to use Coverity and test PostgreSQL regression tests in TimescaleDB
```
Comment (mfreed) [inline CHANGELOG.md]: infrastrcture -> infrastructure

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,48 @@
 # TimescaleDB Changelog
 
+## 0.7.0 (2017-11-21)
+
+**High-level changes**
+* **Initial PostgreSQL 10 support**. TimescaleDB now should work on both PostgreSQL 9.6 and 10. As this is our first release supporting PG10, we look forward to community feedback and testing.
```
Comment (cevian) [inline CHANGELOG.md]: We need a note, probably in bold, that says that this update might take a long time.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,48 @@
...
+* Support for `CLUSTER` on hypertables to recursively apply to chunks
+* Improve constraint handling of edge cases for `DATE` and `TIMESTAMP`
+* Fix `range_start` and `range_end` to properly handle the full 32-bit int space
+* Allow users to specify their desired partitioning function
+* Enforce that the time column contains not NULL values
```
Comment (cevian) [inline CHANGELOG.md]: Enforce NOT NULL constraint on time columns.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,51 @@
 # TimescaleDB Changelog
 
+## 0.7.0 (2017-11-21)
+
+**Please note: This update may take a long time (minutes, even hours) to
+complete, depending on the data size of your database size**
```
Comment (solugebefola) [inline CHANGELOG.md]: "the data size of your database size" -> "your database size"?

**Pull request merged**